app/code/local/Magmodules/Webwinkelconnect/Model/Observer.php

```diff
@@ -159,4 +159,38 @@ public function processInvitationcall($observer)
         }
     }
 
+    /**
+     * @throws Exception
+     */
+    public function sendSyncUrl(Varien_Event_Observer $observer) {
+        if (
+            Mage::getStoreConfig('webwinkelconnect/product_review_invites/enabled')
+            && Mage::getStoreConfig('webwinkelconnect/general/enabled')
+        ) {
+            $helper = Mage::helper('webwinkelconnect');
+            try {
+                $curl = new Varien_Http_Adapter_Curl();
+                $curl->setConfig([
+                    'timeout'   => 10,
+                ]);
+                $url = 'https://dashboard.webwinkelkeur.nl/webshops/sync_url';
+                $data = json_encode([
+                    'webshop_id' => $helper->getShopId(),
+                    'api_key' => $helper->getApiKey(),
+                    'url' => Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_WEB) . 'webwinkelkeur/index/sync',
+                ]);
+
+                $curl->write(Zend_Http_Client::POST, $url, '2', ['Content-Type:application/json'], $data);
+                $response = $curl->read();
+                if (!$response) {
+                    Mage::log(sprintf('(%s) %s',$curl->getErrno(), $curl->getError()));
+                    throw new Exception(sprintf('(%s) %s',$curl->getErrno(), $curl->getError()));
+                }
+                $curl->close();
+            } catch (Exception $e) {
```

Comment:
Generally `catch` statements should be written to catch only exceptions that you expect. In this case that would be a cURL specific exception class, unless Magento doesn't use any.